### PR TITLE
holesky support

### PR DESCRIPTION
**Pull request by lior-blox**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,12 +16,13 @@ require (
 	github.com/stretchr/testify v1.8.4
 )
 
+require github.com/google/go-cmp v0.5.9
```
Comment (GalRogozinski) [inline go.mod:19]: just undo this change

File: go.mod [hunk 2]
```diff
@@ -16,12 +16,12 @@ require (
 	github.com/stretchr/testify v1.8.4
 )
 
+
 require (
 	github.com/btcsuite/btcd/btcec/v2 v2.2.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/google/go-cmp v0.5.9 // indirect
```
Comment (GalRogozinski) [inline go.mod:24]: what I meant before is that go.mod should be unchanged

**Review by GalRogozinski — APPROVED** [review]

**Review by y0sher — APPROVED** [review]

**Review by zippoxer — APPROVED** [review]

**Review by moshe-blox — APPROVED** [review]
